syntax/verilog_systemverilog.vim

```diff
@@ -312,6 +312,14 @@ if index(s:verilog_syntax_fold, "define") >= 0 || index(s:verilog_syntax_fold, "
     " verilogGlobal keywords. This syntax match fixes that:
     syn match verilogGlobal "\<`\(ifn\?def\|e\(els\(e\|if\)\)\|ndif\)\>"
 endif
+if index(s:verilog_syntax_fold, "instance") >= 0 || index(s:verilog_syntax_fold, "all") >= 0
+    syn region verilogFold
+        \ start="^\s*\w\+\(\s*#\s*(\|\s\+\w\+\s*(\)\s*\(//.*\)\?$"
```

Comment:
Yeah your right. For some reason In my head I thought the regex's could span multiple lines but just realised they can't.